akvo/rsr/static/scripts-src/my-reports.jsx

```diff
@@ -180,7 +180,7 @@ function initReact() {
                     );
                 } else {
                     return (
-                        <span key={format.key} />
```

Comment:
:bug: (I can't put the comment where the bug actually is...) Above on line 176 format.displayName should be format.display_name.